rs/embedders/benches/compilation.rs

```diff
@@ -90,6 +90,19 @@ fn generate_binaries() -> Vec<(String, BinaryEncodedWasm)> {
 
     result.push(("open_chat".to_string(), open_chat_wasm));
 
+    // This benchmark uses the open chat user canister which is stored as a
+    // binary file in this repo.  It is generated from the directory
+    // `rust/qrcode` in
+    // https://github.com/dfinity/examples/tree/abk/for-replica-benchmarking
```

Comment:
Comment feels wrong? Copy-pasta?

Reply:
Oh yeah, I missed the first line. But the URL is right.